roborazzi/src/main/java/com/github/takahirom/roborazzi/RobolectricDeviceQualifiers.kt

```diff
@@ -0,0 +1,153 @@
+  const val Device34WQVGAPort = "w264dp-h476dp-normal-long-notround-port-any-ldpi-keyshidden-nonav"
+  const val Device34WQVGALand = "w264dp-h476dp-normal-long-notround-land-any-ldpi-keyshidden-nonav"
+  const val Device37WVGANexusOnePort =
+    "w302dp-h503dp-normal-long-notround-port-any-hdpi-keyshidden-trackball"
+  const val Device37WVGANexusOneLand =
+    "w302dp-h503dp-normal-long-notround-land-any-hdpi-keyshidden-trackball"
+  const val Device37FWVGAsliderPort =
+    "w289dp-h515dp-normal-long-notround-port-any-hdpi-keyshidden-nonav"
+  const val Device37FWVGAsliderLand =
+    "w289dp-h515dp-normal-long-notround-land-any-hdpi-keyshidden-nonav"
+  const val Device4WVGANexusSPort =
+    "w326dp-h544dp-normal-long-notround-port-any-hdpi-keyshidden-nonav"
+  const val Device4WVGANexusSLand =
+    "w326dp-h544dp-normal-long-notround-land-any-hdpi-keyshidden-nonav"
+  const val Device465720pGalaxyNexusPort =
+    "w364dp-h648dp-normal-long-notround-port-any-xhdpi-keyshidden-nonav"
+  const val Device465720pGalaxyNexusLand =
+    "w364dp-h648dp-normal-long-notround-land-any-xhdpi-keyshidden-nonav"
+  const val Device47WXGALand = "w640dp-h360dp-normal-long-notround-land-any-xhdpi-keyshidden-nonav"
+  const val Device47WXGAPort = "w640dp-h360dp-normal-long-notround-port-any-xhdpi-keyshidden-nonav"
+  const val Device51WVGAPort = "w419dp-h699dp-large-long-notround-port-any-mdpi-keyshidden-nonav"
+  const val Device51WVGALand = "w419dp-h699dp-large-long-notround-land-any-mdpi-keyshidden-nonav"
+  const val Device54FWVGAPort = "w424dp-h754dp-large-long-notround-port-any-mdpi-keyshidden-nonav"
+  const val Device54FWVGALand = "w424dp-h754dp-large-long-notround-land-any-mdpi-keyshidden-nonav"
+  const val Device7WSVGATabletLand =
+    "w969dp-h568dp-large-long-notround-land-any-mdpi-keyshidden-nonav"
+  const val Device7WSVGATabletPort =
+    "w969dp-h568dp-large-long-notround-port-any-mdpi-keyshidden-nonav"
+  const val Device101WXGATabletLand =
+    "w1374dp-h859dp-xlarge-long-notround-land-any-mdpi-keyshidden-nonav"
+  const val Device101WXGATabletPort =
+    "w1374dp-h859dp-xlarge-long-notround-port-any-mdpi-keyshidden-nonav"
+
+  // FROM: wear.xml
+  const val WearOSSquareLand = "w186dp-h186dp-small-long-round-land-watch-hdpi-keyshidden-nonav"
+  const val WearOSSquarePort = "w186dp-h186dp-small-long-round-port-watch-hdpi-keyshidden-nonav"
+  const val WearOSRoundLand = "w213dp-h213dp-small-long-round-land-watch-hdpi-keyshidden-nonav"
+  const val WearOSRoundPort = "w213dp-h213dp-small-long-round-port-watch-hdpi-keyshidden-nonav"
+  const val WearOSRoundChinLand =
+    "w249dp-h226dp-small-notlong-notround-land-watch-tvdpi-keyshidden-nonav"
+  const val WearOSRoundChinPort =
+    "w249dp-h226dp-small-notlong-notround-port-watch-tvdpi-keyshidden-nonav"
+
+  // FROM: nexus.xml
+  const val NexusOnePort = "w302dp-h503dp-normal-long-notround-port-any-hdpi-keyshidden-trackball"
```

Comment:
A very personal style thingz I'd probably curate these somewhat, with just ones I expect to be used.

You might also find in practice, people apply the device, then switch portrait and landscape for a single test with "+land"

But all works for me as it includes wear.